libs/librepcb/common/cam/gerbergenerator.h

```diff
@@ -27,6 +27,7 @@
 #include "../fileio/filepath.h"
 #include "../units/all_length_units.h"
 #include "../uuid.h"
+#include "../geometry/vertex.h"
```

Comment:
If only references or pointers to `Vertex` are needed, please replace the include by a forward declaration, just like it was done for the `Path` class. This heavily reduces build time if header files are modified.